### DISCUSSION TITLE
[Question]:  LibreChat v0.7.5-rc2 helm chart not supported for o1

### DISCUSSION THREAD

**Discussion by SharpLu** [danny-avila/LibreChat#4338] [Q&A]
### What is your question?

Hi guys,

I'm used the  LibreChat v0.7.5-rc2 docker image that worked with o1-preview model, but when deploy to kubernates with helm chart , i got the below error, but my kubernates deployment worked all other openAI modesl, except the o1 models got such error, can anyone help with?
![001](https://github.com/user-attachments/assets/e4c0ba1b-402c-4424-ab31-54942c91b2b5)

### More Details

here is the helm chart we used 
https://charts.blue-atlas.de/
and the docker image getting from  danny-avila/librechat

https://github.com/bat-bs/helm-charts/blob/main/charts/librechat/values.yaml#L133

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
This is expected, and is already implemented on latest build. You will need to wait until I cut a release or use the dev builds

**Reply by SharpLu** [reply]
Hi @danny-avila  Thank you so much for the feedback? do you have any approximately estimate when a release will happen?
